js/index.js

```diff
@@ -2,8 +2,151 @@ const modal = document.querySelector('#menuModal');
+    img: 'images/projects/acelerometer.png',
+    technologies: ['C++', 'PHP', 'SQL', 'Eagle'],
+    linkToLive: 'https://github.com/LuisDavidRodriguez',
+    linkToSource: 'https://bitbucket.org/david5048/acelerometrodefinitivo/src/master/',
+  }];
+
+function populateModal(number, arr) {
+  const modalProjectTitle = modalProject.querySelector('h3');
+  const modalProjectList = modalProject.querySelector('ul');
+  const modalProjectImage = modalProject.querySelector('img');
+  const modalProjectDescription = modalProject.querySelector('p');
+  const modalProjectSeeLiveBtn = modalProject.querySelector('#seeLiveBtn');
+  const modalProjectSeeSourceBtn = modalProject.querySelector('#seeSourceBtn');
+
+  const project = arr[number];
+  modalProjectTitle.innerText = project.name;
+  modalProjectDescription.innerText = project.description;
+  modalProjectImage.src = project.img;
+  modalProjectSeeLiveBtn.setAttribute('href', project.linkToLive);
+  modalProjectSeeSourceBtn.attributes.href.nodeValue = project.linkToSource;
+
+  // add dinamically the li elements into the list
+  modalProjectList.innerHTML = '';
+  project.technologies.forEach((item) => {
+    const element = document.createElement('li');
+    element.innerText = item;
+    element.classList.add('sto');
+    modalProjectList.append(element);
+  });
+}
+
+function populateProjects(arr) {
+  const allTheProjects = document.querySelectorAll('.project-container');
+  let iterator = 1;
+
+  allTheProjects.forEach((projectContainer) => {
+    const data = arr[iterator];
+    const title = projectContainer.querySelector('h2');
+    const description = projectContainer.querySelector('p');
+    const list = projectContainer.querySelector('ul');
+
+    title.innerText = data.name;
+    // eslint-disable-next-line prefer-template
+    description.innerText = data.description.substring(0, 100) + '...';
+    projectContainer.style.background = `url(${data.img}) 50% 50% no-repeat`;
+    projectContainer.style.backgroundSize = 'auto 100%';
+
+    list.innerHTML = '';
+    // populate the list with the tecnologies
+    data.technologies.forEach((tec) => {
+      const tecnologie = document.createElement('li');
+      tecnologie.innerText = tec;
+      tecnologie.classList.add('tec');
+      list.append(tecnologie);
+    });
+
+    // eslint-disable-next-line no-plusplus
+    iterator++;
+  });
+}
+
+function populateStories(arr) {
+  const storiesSection = document.querySelector('.stories-container');
+
+  const data = arr[0];
+  const img = storiesSection.querySelector('img');
+  const title = storiesSection.querySelector('h2');
+  const description = storiesSection.querySelector('p');
+  const list = storiesSection.querySelector('ul');
+
+  title.innerText = data.name;
+  description.innerText = data.description.substring(0, 500);
```

Comment:
- [ ] Your description in pop-up window is not similar so kindly fix font and color to match the design
### your design
![his](https://user-images.githubusercontent.com/81727988/179898592-58a3bf80-7f4d-4137-b55b-713bf1a05982.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/179898607-4685b1dc-e190-4af9-9c14-371bae995a2f.png)

Reply:
hello, Solved! 😄